ros_buildfarm/release_job.py

```diff
@@ -84,7 +84,7 @@ def configure_release_jobs(
         for pkg_name in sorted(explicitly_ignored_pkg_names):
             print('  -', pkg_name)
 
-    dist_cache = get_distribution_cache(index, rosdistro_name)
+    dist_cache = _get_and_parse_distribution_cache(index, rosdistro_name, pkg_names)
```

Comment:
Since the returned result is not a `DistributionCache` anymore please rename the variable to something different, e.g. `cached_packages`.

Same below and for modified function arguments.

Reply:
Sure, done.